style.css

```diff
@@ -0,0 +1,24 @@
+#myBtn button {
```

Comment:
Use classes for targeting styling rather than `ID's` as this is considered bad practice.  When you start using frameworks such as Rails, IDs are generated automatically and could change at any time.